src/haxeLanguageServer/Context.hx

```diff
@@ -381,7 +387,10 @@ class Context {
 	function onDidChangeTextDocument(event:DidChangeTextDocumentParams) {
 		final uri = event.textDocument.uri;
 		if (isUriSupported(uri)) {
-			callFileParamsMethod(uri, ServerMethods.Invalidate);
+			if (!invalidated.exists(uri.toString())) {
```

Comment:
Maybe extract these 3 lines into a function so we don't have to duplicate them (or is there a difference I'm missing)?